src/components/NavBar.tsx

```diff
@@ -1,52 +1,31 @@
 import React from "react"
-import { Navbar } from "flowbite-react"
+import Link from "next/link"
 
-function NavTabs({ currentPage, handlePageChange }) {
+function NavBar() {
   return (
-    <Navbar fluid rounded>
-      <Navbar.Brand></Navbar.Brand>
-      <Navbar.Toggle />
-      <Navbar.Collapse>
-        {/* <Navbar.Link
-          href="#home"
-          onClick={() => handlePageChange("Home")}
-          className={currentPage === "Home" ? "nav-link active" : "nav-link"}
-        >
-          Home
-        </Navbar.Link> */}
-        <Navbar.Link
-          href="#about"
-          onClick={() => handlePageChange("About")}
-          className={currentPage === "About" ? "nav-link active" : "nav-link"}
-        >
-          About Me
-        </Navbar.Link>
-        <Navbar.Link
-          href="#projects"
-          onClick={() => handlePageChange("Projects")}
-          className={
-            currentPage === "Projects" ? "nav-link active" : "nav-link"
-          }
-        >
-          Projects
-        </Navbar.Link>
-        {/* <Navbar.Link
-          href="#resume"
-          onClick={() => handlePageChange("Resume")}
-          className={currentPage === "Resume" ? "nav-link active" : "nav-link"}
-        >
-          Resume
-        </Navbar.Link> */}
-        <Navbar.Link
-          href="#contact"
-          onClick={() => handlePageChange("Contact")}
-          className={currentPage === "Contact" ? "nav-link active" : "nav-link"}
-        >
-          Resume & Contact
-        </Navbar.Link>
-      </Navbar.Collapse>
-    </Navbar>
+    <div className="flex justify-center gap-10">
+      <Link href="/services">
+        <button>
+          <span className="font-bold">Services</span>
+        </button>
+      </Link>
+      <Link href="/listings">
+        <button>
+          <span className="font-bold">Listings</span>
+        </button>
+      </Link>
+      <Link href="/properties">
+        <button>
+          <span className="font-bold">Properties</span>
+        </button>
+      </Link>
+      <Link href="#">
+        <button>
+          <span className="font-bold">NavLink4</span>
```

Comment:
![image](https://github.com/ChrisJo93/JdeltaProps/assets/68129965/ec92f9e6-40a7-4003-a78f-e757711fc483)

Can we cut that Navlink4?